templates/download/raspberry-pi/index.html

```diff
@@ -8,175 +8,146 @@
+      <p>Minimum 4GBs of RAM required</p>
+      <hr />
+      <p class="u-responsive-realign">
         <a href="/download/raspberry-pi/thank-you?version={{ releases.latest.full_version }}&amp;architecture=server-arm64+raspi"onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'Raspberry Pi 64-bit - Ubuntu Server', 'eventLabel' : '{{ releases.latest.full_version }}', 'eventValue' : undefined });" class="p-button">Download Ubuntu Server 23.10</a>
+        &nbsp;
+        <span class="u-text--muted">ISO image, 1.1GB</span>
       </p>
     </div>
   </div>
-</section>
-
-<section class="p-strip--light is-shallow">
-  <div class="row u-vertically-center">
-    <div class="col-8 col-medium-3">
-      <h2>Certified for the Raspberry Pi</h2>
-      <p><a href="/certified">Certified devices</a> are tested for reliability and performance, ensuring you have the best out-of-the-box Ubuntu experience. Ubuntu LTS releases are certified on select Raspberry Pi hardware.</p>
-    </div>
-    <div class="col-4 u-align--center u-hide--small u-hide--medium">
-      {{ image (
-        url="https://assets.ubuntu.com/v1/f76dd871-ubuntu-certified.png",
-        alt="",
-        width="150",
-        height="212",
-        hi_def=True,
-        loading="lazy"
-        ) | safe
-      }}
-    </div>
-  </div>
-</section>
-
-<section class="p-strip--light u-no-padding--top">
-  <div class="row u-vertically-center">
-    <div class="col-4 u-align--center u-hide--small u-hide--medium">
-      {{ image (
-        url="https://assets.ubuntu.com/v1/b743f1ee-jj-in-circle.png",
-        alt="",
-        width="250",
-        height="250",
-        hi_def=True,
-        loading="lazy"
-        ) | safe
-      }}
+  <hr class="p-rule is-fixed-width" />
+  <div class="row p-section">
+    <div class="col-6">
+      <h2>Ubuntu {{ releases.lts.full_version }} <abbr title="Long-term support">LTS</abbr></h2>
     </div>
-    <div class="col-8">
-      <h3>Ubuntu {{ releases.lts.full_version }} LTS</h3>
-      <p>The latest Long Term Supported release of Ubuntu with five years of security patching and maintenance until April 2027, extended to ten years with <a href="/pro">Ubuntu Pro</a>.</p>
-      <p>
-        <a href="/download/raspberry-pi/thank-you?version={{ releases.lts.full_version }}&amp;architecture=desktop-arm64+raspi" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Download', 'eventAction' : 'Raspberry Pi 64-bit - Ubuntu Desktop', 'eventLabel' : '{{ releases.lts.full_version }}', 'eventValue' : undefined });" class="p-button u-no-margin--bottom">Download Ubuntu Desktop {{ releases.lts.full_version }} LTS</a> <p><em>* Minimum 2GBs of RAM and 16GB storage required</em></p>
+    <div class="col-6">
+      <p class="u-responsive-realign">The latest Long Term Supported release of Ubuntu with five years of security patching and maintenance until April 2027, extended to ten years with <a href="/pro">Ubuntu Pro</a>.</p>
```

Comment:
Please remove the `u-responsive-realign` from here, it's unnecessary and makes the link have weird spacing.